src/main/java/com/github/sbaudoin/sonar/plugins/yaml/rules/YamlRulesDefinition.java

```diff
@@ -32,17 +32,17 @@
      */
     public static final String RULES_DEFINITION_FOLDER = "org/sonar/l10n/yaml/rules/yaml";
 
-
```

Comment:
I usually add extra lines for readability to separate major sections in my code: statics, constructors, private methods, etc.